tests/apps/core/test_core_api_meetings.py

```diff
@@ -212,6 +212,7 @@ def test_api_meetings_list_authenticated_format(self, mock_token):
                 },
             ],
         )
+        meeting_name = str(meeting.id).replace("-", "")
```

Comment:
Why don't use `meeting.jitsi_name` here, and in the other tests ?

Reply:
Good idea. I can do that if I add a test in `test_models_meetings.py` to secure the format of this computed.attribute. Let me do that!